app/helpers/alaveteli_pro/info_requests_helper.rb

```diff
@@ -16,7 +16,7 @@ def publish_at_options
   end
 
   def embargo_extension_options(embargo)
-    options = embargo_options_from_date(embargo.publish_at)
+    options = embargo_options_from_date(embargo&.publish_at || Date.today)
```

Comment:
I wouldn't do anything about this now as its fine as is, but worth noting that this would have been better as two commits.

1. Allow `InfoRequestsHelper#embargo_extension_options` to accept a `nil` embargo
2. Don't initialise new instances of `Embargo` or `EmbargoExtenstion` in forms

I find that an easier trail of thought to follow than a single "Refactor X" commit.